src/components/challenge/Challenge.js

```diff
@@ -1,11 +1,103 @@
 import React, { Component } from 'react';
 
+const OverviewSection= () => {
```

Comment:
Go ahead and break each of these sections into their own component files. If we do end up wanting to make it super easy for admins to edit this information "in-place", that's gonna definitely require adding some js logic that'll make this file pretty beefy unless we break it up.